addons/l10n_in/models/res_partner.py

```diff
@@ -18,6 +18,7 @@ class ResPartner(models.Model):
             ('deemed_export', 'Deemed Export'),
             ('uin_holders', 'UIN Holders'),
         ], string="GST Treatment")
+    l10n_in_pan = fields.Char(string="PAN")
```

Comment:
it's a pity having such a detailed commit message explaining the field and no help tooltip or comment on it. Is anyone in India supposed to know the meaning of 'PAN' ?

as a reference, here's what we have for the 'TIN' 

> vat = fields.Char(string='Tax ID', index=True, help="The Tax Identification Number. Complete it if the contact is subjected to government taxes. Used in some legal statements.")